src/topojson/topojson.ml

```diff
@@ -203,14 +208,15 @@ module Make (J : Intf.Json) = struct
 
       let base_of_json json = parse_by_arcs json parse_arcs typ
 
-      let to_json ?bbox ?(foreign_members = []) arcs =
+      let to_json ?bbox ?(properties = []) ?(foreign_members = []) arcs =
         J.obj
           ([
              ("type", J.string typ);
              ("arcs", J.array (J.array (J.array J.int)) arcs);
            ]
           @ bbox_to_json_or_empty bbox
-          @ foreign_members)
+          @ foreign_members
+          @ properties)
```

Comment:
These are slightly wrong. The properties are a value associated with the `"properties"` field so you need to construct it similarly to how `"type"` and `"arcs"` are. Something like:

```ocaml
 @ (if properties = [] then [] else [ ("properties", J.obj properties) ])
```